ingestion/src/metadata/ingestion/source/database/bigquery/connection.py

```diff
@@ -107,9 +109,27 @@ def get_tags(taxonomies):
             return policy_tags
 
     def test_tags():
-        taxonomies = PolicyTagManagerClient().list_taxonomies(
-            parent=f"projects/{engine.url.host}/locations/{service_connection.taxonomyLocation}"
-        )
+        taxonomy_project_ids = []
+        if engine.url.host:
+            taxonomy_project_ids.append(engine.url.host)
```

Comment:
I believe that the `engine.url.host` does not contain the GCP ProjectID, and therefore, it might be acceptable to remove this backward compatibility. If the OpenMetadata team agrees with this perspective, please let me know, and I can make the necessary changes in this Pull Request.

Reply:
@civitaspo Thanks for the PR!
I had a question around connection to Bigquery..

how are you connecting to GCP? is it via service account credentials ( json file ) location, cli auth or is it via adding the service account details in UI?

Reply:
@ayush-shah Thank you for your response.
I am not using the UI. Instead, I am specifying a configuration YAML for ingestion/profile in the metadata CLI and executing metadata ingestion and profiling through it.